test/integration_tests/test_select_executor.py

```diff
@@ -88,6 +88,13 @@ def test_should_load_and_select_in_table(self):
         actual_batch.sort()
         expected_batch = list(create_dummy_batches())
         self.assertEqual([actual_batch], expected_batch)
+    
+    def test_should_select_star_in_table(self):
```

Comment:
Gave suggestions for two more test cases.

Reply:
Maybe add test with nested query and join query

Reply:
I will add nested query and join.